src/browser/client-scripts/index.js

```diff
@@ -70,10 +70,11 @@ function prepareScreenshotUnsafe(areas, opts) {
         }
     }
 
-    var viewportWidth = document.documentElement.clientWidth,
-        viewportHeight = document.documentElement.clientHeight,
-        documentWidth = document.documentElement.scrollWidth,
-        documentHeight = document.documentElement.scrollHeight,
+    var mainDocumentElem = util.getMainDocumentElem(),
```

Comment:
If user use command `browser.switchToFrame(<iframe>)` in test then `document.documentElement` references to the document inside frame which has its own dimensions. It can't be used because `getBoundingClientRect` inside iframe returns rect only inside this iframe.

So I should use main document in order to correctly screen element inside iframe.